README.md

```diff
@@ -1,70 +1,202 @@
-# Getting Started with Create React App
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM Done, Thanks